test/Dialect/SV/basic.mlir

```diff
@@ -42,6 +42,37 @@ func @test1(%arg0: i1, %arg1: i1) {
   // CHECK-NEXT:   }
   // CHECK-NEXT: }
 
+  sv.alwaysff "posedge", %arg0, noreset {
+    sv.fwrite "Yo\n"
+  }
+
+  // CHECK-NEXT: sv.alwaysff posedge, %arg0, noreset  {
+  // CHECK-NEXT:   sv.fwrite "Yo\0A"
+  // CHECK-NEXT: }
+
+  sv.alwaysff "posedge", %arg0, syncreset, "posedge", %arg1 {
```

Comment:
This syntax is difficult for me to interpret. I would suggest grouping the clock stuff and the reset stuff with parens. Also, ditch the quotes. Eg:

```
sv.alwaysff (posedge, %arg0), (syncreset, posedge, %arg1) {

Reply:
quotes are optional.
unsure what I think about the parents.

Reply:
Yeah, I'd drop the quotes from the testcase since they aren't needed.  I think that adding the parens is also pretty reasonable.  You can drop the commas, and go with something like:
`sv.alwaysff (posedge: %arg0), (syncreset posedge: %arg1) {`

Or make it more clear that the sync reset binds with the block, like this:

`sv.alwaysff (posedge: %arg0), syncreset posedge: %arg1 {`